PR for blog post -- How I learned Vespa by thinking in Solr

sujitpal opened the pull request:
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

kkraune commented on _posts/2021-02-19-how-i-learned-vespa-by-thinking-in-solr.md:
```suggestion
$ docker exec vespa bash -c "/opt/vespa/bin/vespa-deploy prepare \     
```

kkraune commented on _posts/2021-02-19-how-i-learned-vespa-by-thinking-in-solr.md:
```suggestion
       /opt/vespa/bin/vespa-deploy activate"
```

kkraune commented on _posts/2021-02-19-how-i-learned-vespa-by-thinking-in-solr.md:
```suggestion
Data for my MVP experiment came from the [CORD-19 dataset](https://allenai.org/data/cord-19),
a collection of scientific papers around COVID-19, provided by Allen AI.
Note that my experiment is completely different from the [CORD-19 Search](https://cord19.vespa.ai/) application,
which incidentally is a great example of what you can do with datasets such as CORD-19 and Vespa.
```
correct link - I suppose this was a copy and paste error from the link before

kkraune commented on _posts/2021-02-19-how-i-learned-vespa-by-thinking-in-solr.md:
```suggestion
Vespa also offers its [HTTP Client](https://docs.vespa.ai/en/vespa-http-client.html),
which is a faster and more robust alternative for batch inserts. To use this, the records to be inserted need to be JSON serialized into a flat file, and the path to the flat file passed to the HTTP Client.
```

kkraune reviewed (approved):
Great! a new nitpicks for easier copy and paste into shell + doc link updates

sujitpal commented:
Thanks for the check, I found an additional place where there were smart quotes (Populating index section), also added an image from Unsplash and renamed the post to reflect pub date for 2021-02-24.

sujitpal commented:
Moved the image to the header now, with attribution as first data block, seems to work on the blog page, should probably work with summary page as well?

kkraune commented on _posts/2021-02-24-how-i-learned-vespa-by-thinking-in-solr.md:
```diff
@@ -0,0 +1,200 @@
+layout: post
+title: How I learned Vespa by thinking in Solr
+image: assets/2021-02-19-how-i-learned-vespa-by-thinking-in-solr/albert-WswqR4xIuH8-unsplash.jpg
+date: '2021-02-24'
```
```suggestion
date: '2021-02-24'
author: sujitpal
```
This will display name on front page / blog post

kkraune commented on _config.yml:
```diff
@@ -57,13 +57,19 @@ authors:
     description: "Principal Software Systems Engineer"
+  sujitpal:
+    name: Sujit Pal
+    display_name: Sujit pal
```
```suggestion
    display_name: Sujit Pal
```

kkraune replied:
not sure if you want a capital P or not :-)

kkraune reviewed (commented):
now auto info will display correctly

sujitpal commented:
added avatar image and pointer.

kkraune commented:
Great - all set for merge tomorrow, I will do it then and verify!
